recipes/dftatom/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "dftatom" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/certik/dftatom/archive/v{{ version }}.tar.gz
+  sha256: 7a34311cc2ca5641a2ba81ea4809c972e36ff0e1798498cf6aa9f365ead5ba59
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('cxx') }}
+    - cmake
```

Comment:
Please also add `make` as a build dependency. It is available by default on x86 images but not on aarch64 or ppc64le.

Reply:
Done!